src/Microsoft.DotNet.Build.Tasks/ResolveNuGetPackageAssets.cs

```diff
@@ -302,10 +291,10 @@ private IList<ITaskItem> GetTaskItemsFromLibrary(Library library, ContentPattern
 
             if (group == null)
             {
-                return taskItems;
+                return null;
             }
 
-            foreach (var item in group.Items)
+            foreach (var item in group.Items.Where(i => Path.GetFileName(i.Path) != "_._"))
             {
                 var taskItem = new TaskItem(Path.Combine(Path.GetDirectoryName(library.Path), item.Path.Replace('/', '\\')));
```

Comment:
unrelated to this PR, but can you replace `\\` with `Path.DirectorySeparatorChar`? it should help Unix portability.